src/documents/consumer.py

```diff
@@ -209,6 +209,7 @@ def run_pre_consume_script(self):
         script_env = os.environ.copy()
         script_env["DOCUMENT_SOURCE_PATH"] = original_file_path
         script_env["DOCUMENT_WORKING_PATH"] = working_file_path
+        script_env["TASK_ID"] = task_id
```

Comment:
The [documentation](https://docs.paperless-ngx.com/advanced_usage/) needs to be updated for the new variable, including type of it and the fact it may be None, for the pre and post

Reply:
In which cases can the task_id be None? As per my understanding, the consumption of a new document is always via a celery task?